app/adapters/-intl-adapter.js

```diff
@@ -0,0 +1,40 @@
+import Ember from 'ember';
+import IntlGetResult from 'ember-intl/models/intl-get-result';
+import Locale from 'ember-intl/models/locale';
+import IntlAdapter from 'ember-intl/adapter';
+
+function normalize (fullName) {
+    Ember.assert('Lookup name must be a string', typeof fullName === 'string');
+
+    return fullName.toLowerCase();
+}
+
+export default IntlAdapter.extend({
+    findLocale: function (locale) {
+        if (locale instanceof Locale) {
```

Comment:
It was needed two resolve the following issues:
https://github.com/yahoo/ember-intl/issues/46 (getValue)
https://github.com/yahoo/ember-intl/issues/24 (getValue)
And https://github.com/yahoo/ember-intl/issues/18 (addMessage)